source/extensions/filters/http/aws_request_signing/config.cc

```diff
@@ -46,15 +46,14 @@ SigningAlgorithm getSigningAlgorithm(
   PANIC_DUE_TO_CORRUPT_ENUM;
 }
 
-Http::FilterFactoryCb AwsRequestSigningFilterFactory::createFilterFactoryFromProtoTyped(
-    const AwsRequestSigningProtoConfig& config, const std::string& stats_prefix,
-    Server::Configuration::FactoryContext& context) {
-
-  auto& server_context = context.serverFactoryContext();
+absl::StatusOr<Http::FilterFactoryCb>
+AwsRequestSigningFilterFactory::createFilterFactoryFromProtoTyped(
+    const AwsRequestSigningProtoConfig& config, const std::string& stats_prefix, DualInfo dual_info,
+    Server::Configuration::ServerFactoryContext& context) {
 
   auto credentials_provider =
       std::make_shared<Extensions::Common::Aws::DefaultCredentialsProviderChain>(
-          server_context.api(), makeOptRef(server_context), config.region(),
+          context.api(), makeOptRef(context), config.region(),
```

Comment:
can still keep the server_context as the var. So overall diff will be less.

Reply:
ack